elephant/statistics.py

```diff
@@ -132,12 +135,12 @@ def mean_firing_rate(spiketrain, t_start=None, t_stop=None, axis=None):
 
     if not axis or not found_t_start:
         return np.sum((spiketrain >= t_start) & (spiketrain <= t_stop),
-                      axis=axis) / (t_stop-t_start)
```

Comment:
Hm that's good to know. Neither the pep8 command evoked from terminal nor my IDE (latest intellij with pycharm plugin or emacs) complained about this. In fact it must be auto format... I will consider this with the new PR.